packages/documentation/src/pages/frontend-support-dashboard/lighthouse-performance-report/lighthouse-report.json

```diff
@@ -0,0 +1,210 @@
+{
+  "homepage": {
+    "path": "/",
+    "s3": "https://vetsgov-website-builds-s3-upload.s3-us-gov-west-1.amazonaws.com/lighthouse/homepage.html"
+  },
+  "/auth/login/callback": {
+    "path": "/auth/login/callback",
+    "s3": "https://vetsgov-website-builds-s3-upload.s3-us-gov-west-1.amazonaws.com/lighthouse/auth/login/callback.html"
```

Comment:
@oseasmoran73 can we move this baseURL (`https://vetsgov-website-builds-s3-upload.s3-us-gov-west-1.amazonaws.com/lighthouse/`) into the jsx file? I'm thinking of the situation where we move the files so we don't have to update the entire file

Reply:
Done!